public/pages/CreateSnapshotPolicy/components/CronSchedule/CronSchedule.tsx

```diff
@@ -55,6 +55,25 @@ const CronSchedule = ({
   const [dayOfWeek, setWeek] = useState(initWeek);
   const [dayOfMonth, setMonth] = useState(initMonth);
 
+  // When edit policy is clicked, during the initial render DEFAULT values get passed
```

Comment:
The miss here is thought useState will re-create those values every time

Reply:
Yeah